pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -68,7 +68,7 @@ type managerImpl struct {
 	// config is how the manager is configured
 	config Config
 	// the function to invoke to kill a pod
-	killPodFunc KillPodFunc
+	killPodFunc KillPodFuncAsync
```

Comment:
I have a pending change that removes this line of code - I don't want to completely block this, but I also wanted to land that refactor.

https://github.com/kubernetes/kubernetes/pull/125774/commits/99cd22d00b10b3047343f96dc7931241a05fc344

Just noting here for other reviewers.